tests/robot-tests/tests/libs/public-api-common.robot

```diff
@@ -46,3 +46,22 @@ verify status of API Datasets
     user waits for caches to expire
     ${status_value}=    get text    xpath:(//div[@data-testid="Status"]//dd[@data-testid="Status-value"]//strong)[2]
     should be equal as strings    ${status_value}    ${expected_status}
+    
+user checks value inside the column heading in the Draft version table
+    [Arguments]    ${text}    ${expected_status}
+    user waits for caches to expire
+    ${status_value}=    get text    xpath:(//div[@data-testid="Status"]//dd[@data-testid="${text}"]//strong)[2]
+    should be equal as strings    ${status_value}    ${expected_status}
+
+user checks contents inside the cell value
+    [Arguments]      ${expected_text}     ${locator}
+     ${status_value}=    get text    ${locator}
+    should be equal as strings    ${status_value}    ${expected_text}
+
+user checks row data heading in the Draft version details table
```

Comment:
I think I'd change this to `user checks table headings for Draft version details table`